ReactUI/src/ui/modules/machine-learning/MachineLearning.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+
```

Comment:
A lot of blank lines in imports doesn't have sense if we are not using order according to dependencies.